src/Avalonia.Controls/Primitives/IPopupHost.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using Avalonia.Controls.Primitives.PopupPositioning;
+using Avalonia.VisualTree;
+
+namespace Avalonia.Controls.Primitives
+{
+    public interface IPopupHost : IDisposable
+    {
+        void SetContent(IControl control);
+        IVisual VisualRoot { get; }
```

Comment:
Why is this property needed? It returns `this` in both implementations of this class.

Reply:
It returns `null` for `PopupHost`. It's needed for devtools, I believe.